app/views/components/_email_link.html.erb

```diff
@@ -3,10 +3,15 @@
   data_attributes ||= {}
 %>
 <div class="app-c-email-link">
+  <% if link_title %>
+    <div class="app-c-email-link__title">
```

Comment:
This could be a `p` tag rather than a `div`

Reply:
Done 😸